plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -4426,22 +4390,16 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
             path="/subscriptions/{subscriptionId}/providers/Microsoft.Network/networkVirtualApplianceSkus",
             path_parameters=["subscriptionId"],
             query_parameters=["api-version"],
-            compared_property=lambda r: r["properties"]["vendor"],
-            binding_property=lambda r: r["name"],
+            first_property=lambda r: r["properties"]["vendor"],
+            second_property=lambda r: r["name"],
         )
```

Comment:
This call is not required - it fetches all virtual networks, which have been already fetched and are available in the graph.